MangoAPI.IntegrationTests/IntegrationTestBase.cs

```diff
@@ -38,7 +38,7 @@ protected IntegrationTestBase()
         var configuration = new ConfigurationBuilder().AddJsonFile(appSettingsPath).Build();
 
         const string dbKey = EnvironmentConstants.IntegrationTestsDatabaseUrl;
-        ConnectionString = ConfigurationHelper.TryGetFromEnvironment(dbKey, configuration);
+        ConnectionString = Environment.GetEnvironmentVariable(dbKey) ?? configuration[dbKey];
```

Comment:
Do we really need this get from environment? As I see we can get connection sting just from configuration itself

Reply:
yes

Reply:
Why? We loaded configuration from file, so we are sure that there is a connection string for integration tests